README.md

```diff
@@ -41,6 +41,7 @@ vLLM is flexible and easy to use with:
 
 vLLM seamlessly supports many Huggingface models, including the following architectures:
 
+- Baichuan-7B (`baichuan-inc/Baichuan-7B`)
```

Comment:
Don't we also support 13B and 13B-chat models? It seems there are multiple versions of Baichuan models (e.g., https://huggingface.co/baichuan-inc/Baichuan-13B-Chat).

Reply:
Oh I just noticed that we are not supporting this model at the moment and there's another PR for this. Never mind!